### ISSUE TITLE
duckduckgo: unstyled elements from search "widgets"

### ISSUE THREAD

**Issue by trinkey** [catppuccin/userstyles#948]
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:duckduckgo

### Describe your problem.

when you search for things like "what's my ip", it shows a little area at the top that is unstyled

### Attach screenshots.

all the instances i know of:

![image](https://github.com/catppuccin/userstyles/assets/97406176/5a577609-be65-46c5-a363-81a92e4058c8)

![image](https://github.com/catppuccin/userstyles/assets/97406176/6c934910-008c-40e1-8173-dfe30427e29a)

![image](https://github.com/catppuccin/userstyles/assets/97406176/1d4825d0-ec73-476b-a7f9-c2097fce8e6b)

![image](https://github.com/catppuccin/userstyles/assets/97406176/d70b22cc-a064-4b3f-8bfe-8114b0007dbe)

![image](https://github.com/catppuccin/userstyles/assets/97406176/c4e2fc6f-7705-41b1-bf43-501ab1b3b1bc)

![image](https://github.com/catppuccin/userstyles/assets/97406176/eb6fa5bd-a2ab-4f3e-9d43-ebf85de8899c)

![image](https://github.com/catppuccin/userstyles/assets/97406176/5926760f-c9b7-4246-a8a6-4dc5ad1afede)

![image](https://github.com/catppuccin/userstyles/assets/97406176/8d9123a2-1d33-47c8-97f9-62488d02dadc)

![image](https://github.com/catppuccin/userstyles/assets/97406176/7563e47a-5607-4781-9bbf-108e1864f070)

bonus thing: this box shadow isn't colored properly

![image](https://github.com/catppuccin/userstyles/assets/97406176/d808160e-4783-422c-81cb-a571fcd20a9e)

### What browser(s) are you seeing the problem on?

tested on firefox and chrome

### Any additional comments?

**tldr**: unstyled elements when searching for:
- shorten &lt;url>
- qr code &lt;text>
- what's my ip
- what's my user agent
- css colors
- stopwatch
- html chars
- html &lt;escapable character>
- calendar
